routes/app/app-manager.js

```diff
@@ -66,6 +66,29 @@ exports.getAppByID = function(req, res, next) {
         });
 };
 
+/**
+ * Modify an app
+ */
+exports.modifyApp = function(req, res, next) {
```

Comment:
@nick22891 think you might have missed this one.  Is the app state the only parameter to be changed?